ethers/Cargo.toml

```diff
@@ -0,0 +1,89 @@
+[package]
+name = "ethers"
+authors = ["Georgios Konstantopoulos <me@gakonst.com>"]
+readme = "../README.md"
+description = "A complete Ethereum and Celo Rust library"
```

Comment:
Are we OK with pulling ethers back down a directory? I initially had pulled it up to surface the examples, but seems fine now given we have separate examples crates

Reply:
I think so, also this way it doesn't pull in every folder like `.github` when packaging the `ethers` crate for release